packages/fluentui/react-northstar/src/components/Alert/Alert.tsx

```diff
@@ -212,12 +226,13 @@ class Alert extends AutoControlledComponent<WithAsProp<AlertProps>, AlertState>
           }),
         })}
         {dismissible &&
-          Button.create(dismissAction, {
+          AlertDismissAction.create(dismissAction, {
             defaultProps: () => ({
-              iconOnly: true,
-              text: true,
-              className: Alert.slotClassNames.dismissAction,
-              styles: styles.dismissAction,
+              danger,
+              warning,
+              info,
+              success,
+              variables,
```

Comment:
We have the urgent and oof variables that can be passed on the alert, so we have to pass them down